crates/client/src/api.rs

```diff
@@ -219,9 +225,48 @@ impl Client {
         old_root: DynHash,
         new_root: DynHash,
     ) -> ProofResult<()> {
-        dbg!(old_root);
-        dbg!(new_root);
-        todo!()
+        let old = old_root.clone();
+        let new = new_root.clone();
+        let request = ConsistencyRequest { old_root, new_root };
```

Comment:
Side note: the request types in the API crate should borrow their data so we don't have these unnecessary clones.

I'll file an issue.

Reply:
Filed as https://github.com/bytecodealliance/registry/issues/104